bcipy/helpers/triggers.py

```diff
@@ -165,12 +164,20 @@ def list(cls) -> List[str]:
         """Returns all enum values as a list"""
         return list(map(lambda c: c.value, cls))
 
+    @classmethod
+    def pre_fixation(cls) -> List:

```

Comment:
`List[TriggerType]`

Reply:
This is another annoying case of forward references (https://peps.python.org/pep-0484/#forward-references). There have been several approaches to fixing these, but it seems the team switched from their recommended approach (https://peps.python.org/pep-0563/). The way moving forward is to use `Self`, but this is not implemented until version 3.11 (https://peps.python.org/pep-0673/). It's not clear what the current recommendation is for older python versions. I tried the `from __future__ import annotations` approach but it didn't work well for me with vscode. I ended up with the most basic string-based approach. For a longer discussion, see https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class.